oci_java_image/test.yaml

```diff
@@ -0,0 +1,5 @@
+schemaVersion: 2.0.0
+commandTests:
+  - name: 'test'
+    command: '/src/main/java/com/example/JavaLoggingClient'
```

Comment:
@thesayyn this is where we fail, it says the path doesn't exist even though I checked the image with `dive` and it's there and executable

Reply:
do we have to use command=java and entrypoint=something.jar? we want to use the bash wrapper stub I think